Energy Saver/Pages/Account/Profile.cshtml

```diff
@@ -14,10 +14,21 @@
                      alt="" class="img-rounded img-responsive" />
             </div>
             <div class="col-md-4 text-color">
-                <h3>@Model.UserName</h3>
-                <p>
-                    <i class="glyphicon glyphicon-envelope text-color"></i> @Model.UserEmailAddress
-                </p>
+                <form method="post">
+                    <div class="form-group">
+                        <label asp-for="UserName" class="control-label">User Name</label>
```

Comment:
There could be a button for the option to edit, not editable fields on page load

Reply:
no